src/fireedge/src/server/routes/api/index.js

```diff
@@ -14,34 +14,142 @@
  * limitations under the License.                                            *
  * ------------------------------------------------------------------------- */
 
+const { env } = require('process')
+const multer = require('multer')
 const { messageTerminal } = require('server/utils/general')
-const { getRouteForOpennebulaCommand } = require('server/utils/opennebula')
-const { defaultFilesRoutes, defaultConfigErrorMessage } = require('server/utils/constants/defaults')
+const {
+  genPathResources,
+  getRequestParameters,
+  getRequestFiles,
+} = require('server/utils/server')
+const {
+  defaultWebpackMode,
+  defaultConfigErrorMessage,
+  defaultTmpPath,
+} = require('server/utils/constants/defaults')
+const { writeInLogger } = require('server/utils/logger')
+const { getSunstoneConfig } = require('server/utils/yml')
 
-const filesDataPrivate = []
-const filesDataPublic = []
-defaultFilesRoutes.map(file => {
+genPathResources()
+
+const appConfig = getSunstoneConfig()
+const optsMulter = { dest: appConfig.tmpdir || defaultTmpPath }
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected that your code is accepting requests with significant content length, which could cause excessive network traffic intensity and resource consumption, and can lead to denial of service (DoS). We recommend that you set the `fileSize` &lt;= 8mb. [Learn more](https://cwe.mitre.org/data/definitions/400.html)